diff --git a/os2web_borger_dk_articles.module b/os2web_borger_dk_articles.module
index 3c3b4e8..6b17a45 100644
--- a/os2web_borger_dk_articles.module
+++ b/os2web_borger_dk_articles.module
@@ -290,7 +290,9 @@ function os2web_borger_dk_articles_after_build($form, &$form_state) {
     // form-field except the locked types.
     $data = field_info_instances('node', 'borger_dk_article');
     foreach ($data as $type => $item) {
-      unset($form[$type]);
+      // Fix notices for multilanguage sites
+      // unset($form[$type]);
+      $form[$type] = array('#language' => $form[$type]['#language']);
     }
     unset($form['title']);
 
